include/swift/SIL/SILCloner.h

```diff
@@ -1888,6 +1888,17 @@ void SILCloner<ImplClass>::visitMoveValueInst(MoveValueInst *Inst) {
   recordClonedInstruction(Inst, MVI);
 }
 
+template <typename ImplClass>
+void SILCloner<ImplClass>::visitDropDeinitInst(DropDeinitInst *Inst) {
+  getBuilder().setCurrentDebugScope(getOpScope(Inst->getDebugScope()));
+  if (!getBuilder().hasOwnership()) {
+    return recordFoldedValue(Inst, getOpValue(Inst->getOperand()));
+  }
+  auto *MVI = getBuilder().createDropDeinit(getOpLocation(Inst->getLoc()),
```

Comment:
Did you use git-clang-format here? I am pretty sure this isn't formatted correctly.

Reply:
What in particular about the formatting is bad? It's even identical to `visitMoveValueInst`, so I don't see a problem here.

Reply:
It doesn't look like everyone uses git-clang-format, since Erik wrote this bit of code that I'm cherry-picking.